web/src/lib/name-wranglers.ts

```diff
@@ -0,0 +1,5 @@
+export const getInitials = (name: string): string =>
+	name
+		.split(' ')
+		.slice(0, 2)
```

Comment:
Maybe take the first and last one for people with multiple given names? Idk what the convention is for this or if we will use this.